std/typecons.d

```diff
@@ -3546,17 +3638,25 @@ unittest
         int quack() { return 2; }
         @property int height() { return 20; }
     }
+    struct HumanStructure
+    {
+        int quack() { return 3; }
+        @property int height() { return 30; }
+    }
 
     Duck d1 = new Duck();
     Human h1 = new Human();
+    HumanStructure hs1;
 
     interface Refleshable
     {
         int reflesh();
     }
     // does not have structural conformance
-    static assert(!__traits(compiles, d1.wrap!Refleshable));
-    static assert(!__traits(compiles, h1.wrap!Refleshable));
+    // issue 12993 prevents uncommenting these asserts
```

Comment:
This bug is no longer reproduceable in HEAD, will look into testing with 2.067 release to close it out.